plugins/woocommerce-beta-tester/woocommerce-beta-tester.php

```diff
@@ -17,6 +17,12 @@
 
 defined( 'ABSPATH' ) || exit;
 
+
+if ( defined( 'WP_CLI' ) ) {
```

Comment:
I believe this is the right place to register the CLI. This file is always run and so it ensures that the CLI is available as long as the plugin is active.